lib/rester/rspec.rb

```diff
@@ -96,7 +102,7 @@ def _validate_test_coverage(ex)
 
         missing_contexts.each { |missing_context, _|
           context_group = _find_or_create_child(verb_group, missing_context)
-          context_group.it { is_expected.to include stub_response }
+          context_group.it { is_expected.to deep_include stub_response }
```

Comment:
I guess it's because there's no easy way to test this in rspec. Anyway, it's fixed with the latest commit